name.abuchen.portfolio/src/name/abuchen/portfolio/model/LatestSecurityPrice.java

```diff
@@ -80,4 +82,11 @@ public boolean equals(Object obj)
         LatestSecurityPrice other = (LatestSecurityPrice) obj;
         return super.equals(other) && high == other.high && low == other.low && volume == other.volume;
     }
+
+    @Override
+    @SuppressWarnings("nls")
+    public String toString()
+    {
+        return String.format("%s %,10.2f %,10.2f %,10d", super.toString(), high / Values.Quote.divider(), low / Values.Quote.divider(), volume);
+    }
```

Comment:
```suggestion

@Override
@SuppressWarnings("nls")
public String toString()
{
    String highStr = (high == NOT_AVAILABLE) ? Messages.LabelNotAvailable : String.format("%,10.2f", high / Values.Quote.divider());
    String lowStr = (low == NOT_AVAILABLE) ? Messages.LabelNotAvailable : String.format("%,10.2f", low / Values.Quote.divider());
    String volumeStr = (volume == NOT_AVAILABLE) ? Messages.LabelNotAvailable : String.format("%,10d", volume);

    return String.format("%s %10s %10s %10s", super.toString(), highStr, lowStr, volumeStr);
}
```
You would still have to add the translations to the label, but you could copy them from the [name.abuchen.portfolio.ui.Messages;](https://github.com/portfolio-performance/portfolio/blob/master/name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/Messages.java#L782)
The translations from the resource bundle as well.
It's just an idea...